requirements.txt

```diff
@@ -1,5 +1,4 @@
-#libraries used by our code
 paho-mqtt==2.0.0
-protobuf==4.25.3
-rich==13.7.1
-pydantic==2.6.1
\ No newline at end of file
+protobuf>=4.25.3
```

Comment:
Protobuf must be exact version, I think even bugfixes are not compatible between each other